moveit_core/collision_distance_field/CMakeLists.txt

```diff
@@ -12,9 +12,8 @@ ament_target_dependencies(${MOVEIT_LIB_NAME}
   urdf
```

Comment:
ex: this file still has "ament_target_dependencies(${MOVEIT_LIB_NAME}"

Reply:
same